pkg/cmd/server/certs/mint_allcerts.go

```diff
@@ -0,0 +1,138 @@
+package certs
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/golang/glog"
+	"github.com/spf13/cobra"
+
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/util"
+)
+
+type MintAllCertsOptions struct {
+	CertDir    string
+	SignerName string
+
+	Hostnames util.StringList
+	NodeList  util.StringList
+
+	Overwrite bool
+}
+
+func NewCommandMintAllCerts() *cobra.Command {
+	options := &MintAllCertsOptions{}
+
+	cmd := &cobra.Command{
+		Use:   "mint-all-certs",
+		Short: "Mint all certificates for OpenShift All-In-One",
+		Run: func(c *cobra.Command, args []string) {
+			if err := options.Validate(args); err != nil {
+				fmt.Println(err.Error())
+				c.Help()
+				return
+			}
+
+			if err := options.MintAllCerts(); err != nil {
+				glog.Fatal(err)
+			}
+		},
+	}
+
+	flags := cmd.Flags()
+
+	flags.StringVar(&options.CertDir, "cert-dir", "openshift.local.certificates", "The certificate data directory.")
+	flags.StringVar(&options.SignerName, "signer-name", DefaultSignerName(), "The name to use for the generated signer.")
+
+	flags.Var(&options.Hostnames, "hostnames", "Every hostname or IP you want server certs to be valid for. Comma delimited list")
+	flags.Var(&options.NodeList, "nodes", "The names of all static nodes you'd like to generate certificates for. Comma delimited list")
+	flags.BoolVar(&options.Overwrite, "overwrite", true, "Overwrite existing cert files if found.  If false, any existing file will be left as-is.")
```

Comment:
The actual mint commands should default to true or the commands will disobey you on the command line.